reV/supply_curve/point_summary.py

```diff
@@ -627,6 +622,9 @@ def point_summary(self, args=None):
                 'timezone': self.timezone,
                 }
 
+        if self.offshore is not None:
+            ARGS['offshore'] = self.offshore
```

Comment:
Why do we need to track offshore still?

Reply:
I figured it is definitely worthwhile to have an offshore flag (0 or 1) in the aggregation output table if nothing else than to just filter the offshore sites easily. We dont treat those points any differently but its more of a meta data tracking thing.

Reply:
Ah gotcha so this is just for the sc/agg output table, agreed there!